hmt_escrow/storage.py

```diff
@@ -25,7 +25,8 @@
 
 ESCROW_BUCKETNAME = os.getenv("ESCROW_BUCKETNAME", "escrow-results")
 ESCROW_AWS_ACCESS_KEY_ID = os.getenv("ESCROW_AWS_ACCESS_KEY_ID", "minio")
-ESCROW_AWS_SECRET_ACCESS_KEY = os.getenv("ESCROW_AWS_SECRET_ACCESS_KEY", "minio123")
+ESCROW_AWS_SECRET_ACCESS_KEY = os.getenv("ESCROW_AWS_SECRET_ACCESS_KEY",
+                                         "minio123")
```

Comment:
just imho, 80 characters in one line is something in the past. 120 chars seem fine for all modern devices

Reply:
Alright.

Reply:
I did not change the code due to number of chars to not cause many changes. But I have configured my environment locally to 120 chars.